src/components/Drawer/index.js

```diff
@@ -101,15 +101,17 @@ export default class Drawer extends React.Component {
             )}
           >
             <DrawerContent>
-              {this.renderHeader({
-                toggleDrawer,
-                isOpen,
-                withSpotLight
-              })}
-              {this.renderChildren({
-                toggleDrawer,
-                isOpen
-              })}
+              <div>
```

Comment:
Could you create the separate styled component and add some comments on why do we need this additional wrapper? 

E.g.
```
// Need this wrapper to fix 'some special case'
 const DrawerContentWrapper = styled.div......

```